jenkins/check-for-build.jenkinsfile

```diff
@@ -11,11 +11,10 @@ pipeline {
     }
     triggers {
         parameterizedCron '''
-            H 1 * * * %INPUT_MANIFEST=1.3.5/opensearch-dashboards-1.3.5.yml;TARGET_JOB_NAME=distribution-build-opensearch-dashboards
-            H 1 * * * %INPUT_MANIFEST=2.2.0/opensearch-dashboards-2.2.0.yml;TARGET_JOB_NAME=distribution-build-opensearch-dashboards
+            H/60 * * * * %INPUT_MANIFEST=2.2.0/opensearch-2.2.0.yml;TARGET_JOB_NAME=distribution-build-opensearch
+            H/60 * * * * %INPUT_MANIFEST=2.2.0/opensearch-dashboards-2.2.0.yml;TARGET_JOB_NAME=distribution-build-opensearch-dashboards
```

Comment:
Can you also add test.yaml? We decided to start testing early than to wait till release testing day.
cc: @bbarani

Reply:
@gaiksaya I will add it once we have all the components up.

Thanks.

Reply:
Idea is to fail early and get it running for existing components than waiting for all components and then start running

Reply:
Added now.